hq_superset/utils.py

```diff
@@ -1,3 +1,17 @@
+import sqlalchemy
 
 def get_datasource_export_url(domain, datasource_id):
     return f"a/{domain}/configurable_reports/data_sources/export/{datasource_id}?format=csv"
+
+
+def get_ucr_database():
+    # Todo; get actual DB once that's implemented
+    return db.session.query(Database).filter_by(database_name="HQ Data").one()
+
+
+def create_schema_if_not_exists(schema_name):
```

Comment:
FYI @akashkj @proteusvacuum @kaapstorm   I think we can avoid https://github.com/dimagi/hq_superset/pull/1, let me know if this is missing anything from that PR.

Reply:
LGTM